src/foreach.js

```diff
@@ -1,40 +1,55 @@
+
+import {
```

Comment:
I commented elsewhere, but the `foreach` binding itself may be replaced by knockout-fast-foreach ... but we can still export / make it available.